docs/install-upgrade.md

```diff
@@ -76,9 +144,9 @@ To configure the Operator to watch resources in other namespaces:
    kubectl apply -k config/rbac --namespace <my-namespace>
    ```
 
-5. [Install the operator](#procedure).
+5. [Install the operator](#procedure-using-kubectl).
 
-### Configure the MongoDB Docker Image or Container Registry
+##### Configure the MongoDB Docker Image or Container Registry
```

Comment:
likely a follow-on ticket, but do we want a corresponding section for helm?

Reply:
I wasn't sure if this was configured automatically as part of using Helm for install. If it's needed for Helm we could open a ticket to add this if instructions differ. If they're the same, I could add this to the Helm section. probably need a tech reviewer's input on this.

Reply:
Currently, configuring the registry using helm is not possible.